docs/learn/grammar/numerals.mdx

```diff
@@ -15,130 +15,130 @@ import Numbers599 from '@site/src/markdown/numbers-5-99.mdx';
+For the average Westerner it would make most sense if the word :isv[jedin] were followed by a noun in the singular, and all the remaining numerals by a noun in the plural: :isv[jedin dom], :isv[dva domy], :isv[pet domy], :isv[dvadeset domy], :isv[milion domy]. However, in the Slavic languages things work differently. All numbers higher than 4 are followed by the genitive plural: :isv[pet domov] (:abbr[lit.] „five of houses”). In the case of the numbers 2-4, most languages use the nominative plural in most cases, but some languages, notably Serbo-Croat and Russian, prefer the genitive singular instead. For reasons of clarity, however, it is recommended to use the nominative singular after 1, the nominative plural after 2-4 and the genitive plural after 5 and more: :isv[jedin dom], :isv[dva domy], :isv[četyri domy], :isv[pet domov].
 
 ### Declension of cardinal numbers \{#declension-of-cardinal-numbers}
 
-The easiest solution for the declension of cardinal numbers is simply not to decline them at all. _Dom s tri etažami_ „A house with three floors” is perfectly understandable, even though it may sound a little strange to native speakers. However, a better effect can be achieved by using the following declension patterns:
+The easiest solution for the declension of cardinal numbers is simply not to decline them at all. :isv[Dom s tri etažami] „A house with three floors” is perfectly understandable, even though it may sound a little strange to native speakers. However, a better effect can be achieved by using the following declension patterns:
 
 #### 1 \{#1}
 
-Except for the masculine nominative singular, the word `jedin` „one” is declined like an adjective **\*`jedny`**: m. **`jedin`, `jednogo`**, f. **`jedna`, `jednoj`**, n. `jedno`, etc. In some cases, it can also be used in the plural: :isv[Jedni ljudi prědpočitajut lěto, drugi zimu] „Some people prefer Summer, others Winter”. This is also the case for pluralia tantum: _`jedne dveri`_ „one door”.
+Except for the masculine nominative singular, the word :isv[jedin] „one” is declined like an adjective **\*:isv[jedny]**: :abbr[m.] **:isv[jedin], :isv[jednogo]**, :abbr[f.] **:isv[jedna], :isv[jednoj]**, :abbr[n.] :isv[jedno], etc. In some cases, it can also be used in the plural: :isv[Jedni ljudi prědpočitajut lěto, drugi zimu] „Some people prefer Summer, others Winter”. This is also the case for pluralia tantum: :isv[jedne dveri] „one door”.
```

Comment:
```suggestion
Except for the masculine nominative singular, the word :isv[jedin] „one” is declined like an adjective \*:isv[jedny]: :abbr[m.] :isv[jedin], :isv[jednogo], :abbr[f.] :isv[jedna], :isv[jednoj], :abbr[n.] :isv[jedno], etc. In some cases, it can also be used in the plural: :isv[Jedni ljudi prědpočitajut lěto, drugi zimu] „Some people prefer Summer, others Winter”. This is also the case for pluralia tantum: :isv[jedne dveri] „one door”.
```